.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+closes #issue-this-closes-here
```

Comment:
@jsamoocha this is the pr template that everyone will now see for every pr opened.

Reply:
Ok, looks good! I didn't see this one on your fork, though. Through the link you gave (when removing the `?template=release-pull-request-template.md` param), I saw a template. But it looked like some generic placeholder. 

If it is not working yet and will take more time to fix, maybe we should add the templates in another PR? You clearly described the release process in the docs, and we can always use that as a checklist.